cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -277,7 +277,10 @@ func (m *AwsManager) SetAsgSize(asg *asg, size int) error {
 
 // DeleteInstances deletes the given instances. All instances must be controlled by the same ASG.
 func (m *AwsManager) DeleteInstances(instances []*AwsInstanceRef) error {
-	return m.asgCache.DeleteInstances(instances)
+	if err := m.asgCache.DeleteInstances(instances); err != nil {
+		return err
+	}
+	return m.forceRefresh()
```

Comment:
It feels like we should either have a log message here stating that a refresh is being forced or update the message in `forceRefresh` as currently it could be confusing for users with the log message
in `forceRefresh` stating: `"Refreshed ASG list, next refresh after %v", m.lastRefresh.Add(refreshInterval))`